src/tests/functional/plugin/conformance/test_runner/conformance_infra/src/read_ir_test/read_ir.cpp

```diff
@@ -228,6 +230,83 @@ void ReadIRTest::SetUp() {
             }
         }
     }
+
+#ifdef ENABLE_CONFORMANCE_PGQL
+    // Updating data in runtime. Should be set before possible call of a first GTEST status
+    auto pgLink = this->GetPGLink();
+    if (pgLink) {
+        if (this->targetDevice == "CPU") {
```

Comment:
Lets do not use hardcode in conformance. My suggestion is to report an architecture && full device name (we can get it from ov runtime -> ) in all cases as attribute of device field
I understand that full device name and architecture could be different, but it is more flexible solution
The example of source code:
```
            supported_metrics = core.get_property(target_device, SUPPORTED_PROPERTIES)
            if FULL_DEVICE_PROPERTY in supported_metrics:
                if core.get_property(target_device, FULL_DEVICE_PROPERTY) != core.get_property(device, FULL_DEVICE_PROPERTY):
                    logger.warning(f'Device {device} is different {FULL_DEVICE_PROPERTY} with {target_device} ( : {core.get_property(device, FULL_DEVICE_PROPERTY)} : {core.get_property(target_device, FULL_DEVICE_PROPERTY)} )')
                    continue
            if DEVICE_ARCHITECTURE_PROPERTY in supported_metrics:
                if not core.get_property(target_device, DEVICE_ARCHITECTURE_PROPERTY) != core.get_property(device, DEVICE_ARCHITECTURE_PROPERTY):
                    logger.warning(f'Device {device} is different {DEVICE_ARCHITECTURE_PROPERTY} with {target_device} ( : {core.get_property(device, DEVICE_ARCHITECTURE_PROPERTY)} : {core.get_property(target_device, DEVICE_ARCHITECTURE_PROPERTY)} )')
                    continue
```

Reply:
Let me check what will I get using this way... It might be a better way, but it doesn't solve DGPU detection way.

Reply:
It isn't a way for correct architecture detection. CPU plugin doesn't have corresponding supported property.

Correct place to put detailed information about a host - hosts/hosts_info table.